apps/condo/domains/user/oidc/configuration.js

```diff
@@ -52,6 +52,9 @@ module.exports = function createConfiguration (context) {
         cookies: {
             // TODO(pahaz): take it from .env!
             keys: ['some secret key', 'and also the old rotated away some time ago', 'and one more'],
+            short: {
+                sameSite: 'None',
```

Comment:
You also need to set secure attribute. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite#none

Reply:
This option doesn't exist for oidc-provider

https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#cookiesshort